packages/flutter/lib/src/widgets/dismissible.dart

```diff
@@ -256,6 +279,35 @@ class DismissUpdateDetails {
   final double progress;
 }
 
+/// Details for [AcceptDismissCallback].
+///
+/// See also:
+///
+///   * [Dismissible.shouldDismiss], which receives this information.
+class AcceptDismissDetails {
+  /// Create a new instance of [AcceptDismissDetails].
+  const AcceptDismissDetails({
+    this.direction = DismissDirection.horizontal,
+    this.reached = false,
+    this.progress = 0.0,
+  });
+
+  /// The direction that the dismissible is being dragged.
+  final DismissDirection direction;
+
+  /// Whether the dismiss threshold is currently reached.
+  final bool reached;
+
+  /// The offset ratio of the dismissible in its parent container.
+  ///
+  /// A value of 0.0 represents the normal position and 1.0 means the child is
+  /// completely outside its parent.
+  ///
+  /// This can be used to synchronize other elements to what the dismissible is doing on screen,
+  /// e.g. using this value to set the opacity thereby fading dismissible as it's dragged offscreen.
+  final double progress;
```

Comment:
I think it'd be nice to have a `velocity` member here too, and perhaps some others:

```dart
/// (passed directly from the [DragEndDetails])
final Velocity flingVelocity;

/// The component of the velocity parallel to the [DismissDirection].
/// A fling away from the center will have a positive value.
double get velocity {
  // copy some logic over from _describeFlingGesture()
}

/// The angle between the direction of the fling and the [DismissDirection].
double get angle {
  // ...
}

/// Similar to [progress], but the value is presented in logical pixels
/// rather than as a percentage of the parent's width.
/// (We're already computing this value, so we might as well use it here!)
final double dragExtent;
```

Reply:
Didn't managed to make it work for `angle`
The values returned from `DragEndDetails` are not suffisant to compute a drag angle.
The Velocity only have a value for the axis used by DismissDirection

Reply:
That's unfortunate to hear, but thanks for letting me know.

With this in mind, I think it would be ideal to remove the `Velocity` object, since that information is already accessible via the `double` velocity and `DismissDirection`.